src/components/tables/Table.tsx

```diff
@@ -264,4 +263,11 @@ export class Table extends React.Component<ITableProps> {
                 </tbody>
             );
     }
+
+    private shouldShowTableBody(): boolean {
+        const tableData = this.props.tableCompositeState.data || this.props.initialTableData;
+
+        return !this.props.tableCompositeState.isInError
+            && !!(tableData.displayedIds.length || this.props.tableCompositeState.isLoading || this.isInitialLoad);

```

Comment:
`!_.isEmpty(tableData.displayedIds)` is safer than `tableData.displayedIds.length`,but maybe `tableData.displayedIds` never is undefined in that case thought. Very minor, your call.

Reply:
will never be undefined 👍

Reply:
but you're right 😄